lib/filterx/filterx-error.h

```diff
@@ -39,6 +39,7 @@ typedef struct _FilterXError
 } FilterXError;
 
 void filterx_error_clear_error(FilterXError *error);
+EVTTAG *filterx_error_format_error(FilterXError *error);
```

Comment:
IMO `error` suffices are not necessary at this point, I would recommend to call this `filterx_error_format()`, which is basically `FilterXError::format()`.

Same applies for the rest of the functions here.